docs/UserGuide.md

```diff
@@ -152,7 +228,20 @@ Format: `edit -t <task_index> <task_name>`
 Example of usage: 
 * `edit -t 3 revise for CS2113`
 
-Note: You can find the task indexes with `list -t`  
+**Note:** <br>
+ You can find the task indexes with `list -t`  
+
+Example of output:
+```
+Command: 
+⋗	edit -t 1 buy stuff
+════════════════════════════════════════════════════════════════════════════════
+Your task has been edited successfully.
+Your task before editing - 'go running [x]'
```

Comment:
A little nit-picky but should it be 'read a book [x]', for continuity from "add -t" section?